src/Paket.Core/Packaging/PackageProcess.fs

```diff
@@ -171,11 +171,10 @@ let Pack(workingDir,dependenciesFile : DependenciesFile, packageOutputPath, buil
             | _ -> true)
         |> Array.map (fun (projectFile,templateFile') ->
             allTemplateFiles.Remove(templateFile'.FileName) |> ignore
-
-            let merged = merge buildConfig buildPlatform version specificVersions projectFile templateFile'
+            let merged = lazy (merge buildConfig buildPlatform version specificVersions projectFile templateFile')
```

Comment:
Idea of this fix is here - make "dangerous" operation lazy. 
So exception only occur if project will actually be needed for packing.